src/app.rb

```diff
@@ -2,59 +2,10 @@
 require './teacher'
 require './book'
 require './rental'
+require './menu'
 
-# Menu blueprint
-# rubocop:disable Metrics/CyclomaticComplexity
 # rubocop:disable Metrics/MethodLength
```

Comment:
- [ ] You can now remove this disabling of rubocop since the method is no longer very long